tests/store_helpers.py

```diff
@@ -60,6 +61,33 @@ def ensure_hk_bundle(store: HousekeeperAPI, bundle_data: dict, include: bool = F
 
         return _bundle
 
+    @staticmethod
+    def format_hk_bundle_dict(bundle_name, files: list[Path], all_tags: list[list[str]]) -> dict:
+        """Creates the dict representation for a housekeeper bundle with necessary values set."""
+        return {
+            "name": bundle_name,
+            "created_at": datetime.now(),
+            "expires_at": datetime.now(),
+            "files": [
+                {
+                    "path": file.as_posix(),
+                    "tags": tags,
+                    "archive": False,
+                }
+                for file, tags in zip(files, all_tags)
+            ],
+        }
+
+    @staticmethod
+    def quick_hk_bundle(
+        bundle_name, files: list[Path], store: HousekeeperAPI, tags: list[list[str]]
+    ):
+        """Adds a bundle to housekeeper with the given files and tags."""
+        bundle_data: dict = StoreHelpers.format_hk_bundle_dict(
+            bundle_name=bundle_name, files=files, all_tags=tags
+        )
+        return StoreHelpers.ensure_hk_bundle(store=store, bundle_data=bundle_data)
+

```

Comment:
This is neat! This does need the order to be correct when passing the files and tags and kind of confused me when you used it in the test above as I was not certain which files got which tags. Would a more elaborate docstring help?

Reply:
Good point. Adding a more verbose doc string